app/validation/validator.py

```diff
@@ -2193,6 +2202,12 @@ def _get_block_ids(json_to_validate):
 
         return block_ids
 
+    @staticmethod
+    def _validate_url(url):
+        parsed_result = urlparse(url)
+
+        return parsed_result.scheme and parsed_result.netloc
```

Comment:
A FDQN is enough?. Even without the scheme, the URL would be valid.

Reply:
This is to validate a url not a FQDN. It isn't enough because it wouldn't specify the scheme? The design system requires a full url to be able to pull the data.